### DISCUSSION TITLE
역 콘웨이의 법칙(Inverse Conway's Law) 에 대해서 다시 한번 찾아 봤습니다

### DISCUSSION THREAD

**Discussion by TaeHyoungKwon** [ThinkAboutSoftware/AcademicConference#327] [General]
서치를 해봤을 때, 역 콘웨이의 전략(Inverse Conway Maneuver), 역 콘웨이의 법칙(Inverse Conway's Law) 둘다 같은 뜻으로 사용되는 것으로 보입니다

아래 내용들에 따르면, 조직에서도 마이크로서비스의 장점을 그대로 가져가기 위해서, 역 콘웨이의 법칙을 활용해 아키텍쳐(마이크로서비스)를 기준으로 조직구조를 만들어간다고 제가 상상한 뇌피셜과 비슷한 맥락에서 이해해볼 수 있을 것 같습니다

-----

[수잔 파울러의 역콘웨이의 법칙에 대한 설명](https://johngrib.github.io/wiki/Conway-s-law/#%EC%88%98%EC%9E%94-%ED%8C%8C%EC%9A%B8%EB%9F%AC)

![스크린샷 2023-06-03 오전 1 53 03](https://github.com/ThinkAboutSoftware/AcademicConference/assets/18538418/e7a5e4fd-4d6a-4eaa-92a9-0c6ef1064e7b)

[실용주의 프로그래머](https://johngrib.github.io/wiki/Conway-s-law/#%EC%8B%A4%EC%9A%A9%EC%A3%BC%EC%9D%98-%ED%94%84%EB%A1%9C%EA%B7%B8%EB%9E%98%EB%A8%B8%EC%97%90%EC%84%9C-%EB%A7%90%ED%95%98%EB%8A%94-%EC%BD%98%EC%9B%A8%EC%9D%B4%EC%9D%98-%EB%B2%95%EC%B9%99)

![스크린샷 2023-06-03 오전 1 58 30](https://github.com/ThinkAboutSoftware/AcademicConference/assets/18538418/5140b272-54ec-454b-ad13-5bb1afefe2e7)

[팀 토폴로지](https://www.aladin.co.kr/shop/wproduct.aspx?ItemId=258906450)

![스크린샷 2023-06-03 오전 1 58 13](https://github.com/ThinkAboutSoftware/AcademicConference/assets/18538418/a8388731-2f89-4a76-9f61-674bddd997c0)

**Comment by TaeHyoungKwon** [comment]
작년에 사두고 안본 팀 토폴로지 책이 콘웨이의 법칙 맥락을 이해하고 나니, 이 새벽에 너무 재밌게 잘 읽히네요 ㅎㅎ 다음 책모임 때는 팀 토폴로지 책을 한번 추천해봐야겠습니다 😄

**Reply by jongfeel** [reply]
실용주의 프로그래머에서도 설명은 되어 있었군요. 여러 책에 언급된 걸 보면 태형님이 주장했던 내용이 뇌피셜(?)은 아닌 것 같아 보입니다 ㅎㅎ
